src/main/java/seedu/address/model/util/SampleDataUtil.java

```diff
@@ -20,22 +22,34 @@
     public static Person[] getSamplePersons() {
         return new Person[] {
             new Person(new Name("Alex Yeoh"), new Phone("87438807"), new Email("alexyeoh@example.com"),
-                new Address("Blk 30 Geylang Street 29, #06-40"),
+                new Year("1"),
+                new Major("cs"),
```

Comment:
I'm not sure about this as I can't test it but does creating a new major with lower case letters work? I thought it was case sensitive

Reply:
Ah, good catch. Will fix.